### DISCUSSION TITLE
Analysed scene cuts to WAV embedded cues

### DISCUSSION THREAD

**Discussion by cristianvogel** [mifi/lossless-cut#1161] [Ideas]
I am a sound designer. I often get short video rushes and need to edit sound to it. 
I thought Lossless-cut could be a great tool to save me having to 'recut' the video scene with markers in the DAW. 

My idea is to create a WAV only export, that embeds the scene cut points as Cue markers in the cue markers WAV header chunk. 

https://www.recordingblogs.com/wiki/cue-chunk-of-a-Wave-file

**Comment by mifi** [comment]
Hi. I'm not sure if this is possible to do with ffmpeg
